### DISCUSSION TITLE
bug  in fzf :Blines

### DISCUSSION THREAD

**Discussion by rinetd** [arcticicestudio/nord#201] [Help & Support]
![default](https://user-images.githubusercontent.com/15907353/34344785-559f513a-ea24-11e7-9cc9-fe054d5317ee.png)

**Comment by arcticicestudio** [comment]
Can you please add a description of what exactly the problem is? It looks like your're refering to a problem with the [fzf.vim][fzfv] plugin, but without more information about the problem and your setup/configuration I've no chance to reproduce and maybe fix the problem.

Also if this only occurs for the *fzf.vim* plugin it might be a plugin-specifc styling group which is not bug within the Nord theme. The theme does currently not provide explicit support for the plugin.

[fzfv]: https://github.com/junegunn/fzf.vim

**Comment by arcticicestudio** [comment]
@rinetd Is the problem still existent or have you been able to solve it? If yes, can you please provide a description about the solution so I can perhaps add it to the documentation or add support for the _fzf.vim_ plugin.

**Comment by rinetd** [comment]
now,I have no idea about it . 
i just use nova instead

![nord](https://user-images.githubusercontent.com/15907353/40914573-4c42a1ca-682b-11e8-8dcf-96022cb1a546.png)

![nova](https://user-images.githubusercontent.com/15907353/40914469-df7d82da-682a-11e8-9dd9-f725ccfab163.png)

 " colorscheme nord
   colorscheme nova

**Comment by svengreb** [comment]
Thank you for your patience @rinetd! 🙏🏼
It‘s been a while [since I had free time to focus more on Nord, and my open source projects in general,](https://github.com/orgs/nordtheme/discussions/183#user-content-todays-scope) and invest time in this issue due [the work-life balance](https://github.com/orgs/nordtheme/discussions/183#work-life-balance). 

I recently published the first [“Northern Post — The state and roadmap of Nord“ announcement](https://github.com/orgs/nordtheme/discussions/183) which includes all details about the plans and future of the _Nord_ project, including [the goal of catching up with the backlog](https://github.com/orgs/nordtheme/discussions/183#catching-up-on-the-backlog). This issue is part of the backlog and therefore I want to triage and process it to get one step closer to a “clean state“.

Due to the scope of this issue of not being “purely technical“, or being related to the project's code at all, I‘ll convert it into a discussion for [Nord‘s central community hub on GitHub](https://github.com/orgs/nordtheme/discussions). This way this repository is easier to maintain the project by separating conversations about the actual code and support topics while also making help requests from users more visible and supportable by the community.

Also since this issue has not been updated and you mentioned that you switched I‘ll also close it.
